### PR TITLE
LANG: python 3.9 update (amplicon)

### PR DESCRIPTION
**Pull request by lizgehret**

### CODE REVIEW

File: 2024.5/amplicon/passed/seed-environment-conda.yml [hunk 1]
```diff
@@ -4,18 +4,19 @@ channels:
...
 dependencies:
-- biom-format=2.1.14
-- h5py=3.8.0
-- hdf5=1.14.0
-- mafft=7.520
-- matplotlib=3.6.0
+- biom-format=2.1.15
+- h5py=3.11.0
+- hdf5=1.14.3
+- mafft=7.525
```
Comment (lizgehret) [inline 2024.5/amplicon/passed/seed-environment-conda.yml]:
```suggestion
- mafft=7.520
```

**Comment by ebolyen** [comment]
related skbio issue, will update q2-diversity-lib and q2-diversity as needed: https://github.com/scikit-bio/scikit-bio/issues/2014

**Comment by ebolyen** [comment]
Looks like https://github.com/qiime2/q2-diversity-lib/tree/Language-python39-update was based off an old branch. We just need to merge `dev` back into it and re-run from the top (it should recycle builds up to q2-div-lib).

File: 2024.5/amplicon/passed/seed-environment-conda.yml [hunk 2]
```diff
@@ -42,13 +43,14 @@ dependencies:
...
 - rescript=2024.5.0.dev0+3.g6e87cfe
-- scikit-bio=0.5.8
-- scikit-learn=0.24.1
-- scipy=1.10.0
-- seaborn=0.12.2
-- unifrac-binaries=1.3.2
+- scikit-bio=0.6.0
+- scikit-learn=1.4.2
+- scipy=1.13.0
+- seaborn=0.13.2
```
Comment (lizgehret) [inline 2024.5/amplicon/passed/seed-environment-conda.yml]:
```suggestion
- seaborn=0.12.2
```

**Comment by lizgehret** [comment]
> Looks like https://github.com/qiime2/q2-diversity-lib/tree/Language-python39-update was based off an old branch. We just need to merge `dev` back into it and re-run from the top (it should recycle builds up to q2-div-lib).

updated here: https://github.com/qiime2/q2-diversity-lib/pull/64

**Comment by lizgehret** [comment]
🥳🥳🥳

**Pull request closed without merging**